Makefile

```diff
@@ -9,10 +9,10 @@ install: requirements.txt
 	$(PYTHON) -m pip install -r requirements.txt
 
 client: install
-	.venv/bin/python3.11 src/client.py
+	.venv/bin/python3.12 src/client.py
```

Comment:
Is this fine for you @tonyhuang12138?

Reply:
I am still a little confused about how to correctly set this up. From what I could observe, running make install creates a .venv/bin directory with python3, and whatever the current python3 version you have. For me, that is python 3.9. So the executables I have are python3 and python3.9. This means that I cannot run the script with 3.12. 

Does this imply that we require the user to have installed python 3.12? If so, should we add this in the `Makefile`? Or should we add in `requirements.txt`

Reply:
Nvm forget about `requirements.txt`, I realized they are python packages only.

Reply:
Discussed via chat and added to tickets.